src/main/resources/static/js/subscription.js

```diff
@@ -30,6 +30,8 @@ jQuery(document).ready(function () {
                 backendStatus = true;
             },
             error: function (XMLHttpRequest, textStatus, errorThrown) {
+                var jsonObj = JSON.parse(XMLHttpRequest.responseText);
```

Comment:
Please check indention here, tabs is used and not spaces.

Reply:
no, its correct number of spaces.